src/screens/CreatePost/index.js

```diff
@@ -89,7 +90,7 @@ const CreatePost = () => {
   const sheetBackRef = React.useRef();
 
   const [typeUser, setTypeUser] = React.useState(false);
-  const {headerTitle, initialTopic, isInCreatePostTopicScreen, anonUserInfo} =
+  const {headerTitle, initialTopic, isInCreatePostTopicScreen, anonUserInfo, setSelectedTopic} =
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `setSelectedTopic` to the destructured assignment from `useCreatePostHook` is a good change if it's being used later in the code. However, it's important to ensure that `setSelectedTopic` is properly defined and exported from `useCreatePostHook`.
```diff
+ const {headerTitle, initialTopic, isInCreatePostTopicScreen, anonUserInfo, setSelectedTopic} =
     useCreatePostHook(typeUser);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->